lib/nordigen-ruby.rb

```diff
@@ -26,15 +26,8 @@ def initialize(secret_id:, secret_key:)
             @requisition = RequisitionsApi.new(client=self)
         end
 
-        def request(params = nil)
```

Comment:
This is just setting up a request function, not running the request, and so Faraday doesn't allow adding params here as far as I can tell.